apps/examples/testcase/Makefile

```diff
@@ -60,11 +60,13 @@ include $(APPDIR)/Make.defs
 APPNAME = tc
 FUNCNAME = $(APPNAME)_main
 THREADEXEC = TASH_EXECMD_ASYNC
+CPPEXT ?= .cpp
 
 # testcase example
 
 ASRCS =
 CSRCS =
+CXXSRCS =
```

Comment:
Don't need to initialize CPPSRCS ? At the below, CPPSRCS is used.

Reply:
yes, i will change that

Reply:
I changed it all to CXX.